code/game/machinery/computer/robot.dm

```diff
@@ -204,7 +204,7 @@
 		robot["master_ai"] = R.connected_ai ? R.connected_ai.name : "None"
 		robot["hackable"] = 0
 		//Antag synths should be able to hack themselves and see their hacked status.
-		if(operator && istype(operator, /mob/living/silicon) && (operator.mind.special_role && operator.mind.original == operator))
+		if(operator && istype(operator, /mob/living/silicon/robot) && (operator.mind.special_role && operator.mind.original == operator) && (operator == R))

```

Comment:
Might as well change that istype check to isrobot(operator)

https://github.com/PolarisSS13/Polaris/blob/125e654fe0c9e34f7615025234778fbd9aedf55a/code/_macros.dm#L32